src/test/java/de/tum/in/www1/artemis/ManagementResourceIntegrationTest.java

```diff
@@ -0,0 +1,123 @@
+package de.tum.in.www1.artemis;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.config.audit.AuditEventConverter;
+import de.tum.in.www1.artemis.domain.PersistentAuditEvent;
+import de.tum.in.www1.artemis.service.AuditEventService;
+import de.tum.in.www1.artemis.service.feature.Feature;
+import de.tum.in.www1.artemis.util.DatabaseUtilService;
+import de.tum.in.www1.artemis.util.RequestUtilService;
+
+public class ManagementResourceIntegrationTest extends AbstractSpringIntegrationTest {
+
+    @Autowired
+    DatabaseUtilService database;
+
+    @Autowired
+    RequestUtilService request;
+
+    @SpyBean
+    AuditEventService auditEventService;
```

Comment:
please do not use a SpyBean here. This significantly increases test execution costs and we absolutely want to avoid it for simple cases in which we can easily avoid this!